superset/assets/javascripts/modules/superset.js

```diff
@@ -141,10 +141,11 @@ const px = function () {
         let msg = '';
         if (!xhr.responseText) {
           const status = xhr.status;
-          msg += 'An unknown error occurred. (Status: ' + status + ')';
           if (status === 0) {
             // This may happen when the worker in gunicorn times out
-            msg += ' Maybe the request timed out?';
+            msg += 'Could not reach server';
```

Comment:
should we give some sort of action here with the error message, like refreshing the page? 

`Sorry, the server could not be reached. Please refresh your page to try again.`